### DISCUSSION TITLE
refund

### DISCUSSION THREAD

**Discussion by tianheng2017** [vercel/community#2287] [Help] [answered]
### Summary

I was charged a monthly fee of $20 for an unexpected service. This was completely unexpected. It sucks. Can I apply for a refund for this?
![Screenshot_20230504_045451](https://user-images.githubusercontent.com/29515610/236047990-2e914365-f4c9-4b22-a53b-6de031e47591.jpg)

**Comment by tianheng2017** [comment]
I didn't know about this service before, but I was surprised when I received the bill, and I don't know when it was started

**Comment by amyegan** [comment, marked as answer]
I'm sorry you had an unexpected charge, @tianheng2017. Please [submit a case](https://vercel.com/help#issues) to the Customer Success Team for billing support 🙏
